code/modules/gear_presets/synths.dm

```diff
@@ -451,6 +451,53 @@
 
 	survivor_variant = ENGINEERING_SURVIVOR
 
+/datum/equipment_preset/synth/upp/upp_synth
+	name = "Survivor - Synthetic - UPP Synth"
+	flags = EQUIPMENT_PRESET_EXTRA
+	assignment = JOB_UPP_COMBAT_SYNTH
+	rank = JOB_SURVIVOR
+	faction = FACTION_UPP
+	faction_group = list(FACTION_UPP, FACTION_SURVIVOR)
+	skills = /datum/skills/colonial_synthetic
+	paygrade = "SYN"
+	idtype = /obj/item/card/id/dogtag
+	role_comm_title = "UPP 173Rd RECON Syn"
+
+/datum/equipment_preset/synth/upp/upp_synth/load_name(mob/living/carbon/human/new_human, randomise)
+	var/random_name
+	if(prob(10))
+		random_name = "[capitalize(randomly_generate_chinese_word(2))]"
+	else if(new_human.gender == MALE)
+		random_name = "[pick(first_names_male_upp)]"
+	else
+		random_name = "[pick(first_names_female_upp)]"
+
+	new_human.change_real_name(new_human, random_name)
+
+/datum/equipment_preset/synth/upp/upp_synth/load_gear(mob/living/carbon/human/new_human)
+
+	new_human.allow_gun_usage = TRUE
```

Comment:
While I'm not a balance reviewer, I can tell you with confidence a survivor combat synth has a next to zero chance of being included, even if it is a rare spawn.